src/can-observable-array.js

```diff
@@ -60,6 +62,24 @@ class ObservableArray extends MixedInArray {
 		return new this(...array);
 	}
 
+	static of (type) {
+		const name = `ObservableArrayOf${canReflect.getName(type)}`;
```

Comment:
Maybe use `<` and `>` in the name like `ObservableArrayOf<Todos>`

Reply:
Sure, yeah I like that